### ISSUE TITLE
no work 

### ISSUE THREAD

**Issue by zacxl** [SavoVuksan/GodotTemplateMenus#2]
Parser Error: Identifier "MenuTemplateManager" not declared in the current scope.

**Comment by uneuro** [comment]
same issue here

**Comment by uneuro** [comment]
resolved with set:

 [autoload]

MenuTemplateManager="*res://addons/EasyMenus/Nodes/menu_template_manager.tscn"

**Comment by hsandt** [comment]
Yup, or add it via the Project Settings popup > Autoload tab, enabled.

@zacxl Could you rename the issue title to contain the error message (or at least part of it)?

EDIT: that said, you can also close the issue after that since the autoload explanation has been added to the README section "How to use" (in fact since 10 Nov, 2022)
